src/components/navigationBar/MenuItem.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import theme from "../../theme";
+import { Flex } from "../container";
+import { ExtendedPopover } from "../extendedPopover";
+import { ExtendedTooltip } from "../extendedTooltip";
+import { SVGIcon } from "../svgicon";
+import { Trigger } from "./components/Trigger";
+import { StyledTriggerWrapper } from "./styles";
+import { Item } from "./types";
+import { useMenuBarConfiguration } from "./utils/hooks";
+
+export function MenuItem({ caption, icon, children }: Item) {
```

Comment:
Some misalignment with the naming of types?

```suggestion
export function MenuItem({ caption, icon, children }: MenuItemProps) {
```